deployment/dockerdeploy/deployer.go

```diff
@@ -556,7 +557,7 @@ func (d *Deployer) addRemoveNodes(
 		}
 
 		setupNodeOpts = append(setupNodeOpts, &clustercontrol.AddNodeOptions{
-			ServerGroup: "0",
+			ServerGroup: nodeGrp.ServerGroup,
```

Comment:
Previously this `"0"` was kind of hard-coded, and it represents UUID of the group. Now `AddNodeOptions.ServerGroup` is the name of the group, not its UUID, and it will take it from the YAML config